update example (#1)
DevExpress-Examples/winforms-map-use-openstreetmap-search-service: Readme.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T629598)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Map for WinForms - Use the OpenStreetMap Search Service

> This example demonstrates how to add an image layer with map tiles from the OpenStreetMap service and an information layer that searches for a place on the map with the means of the OpenStreetMap search service.

Reviewer: ```suggestion
This example demonstrates how to add an image layer with map tiles from the OpenStreetMap service, and an information layer uses the OpenStreetMap search service to locate a place on the map.
```

![resulting map](Images/resulting-map.png)

The example uses the following classes:

* [`ImageLayer`](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.ImageLayer) - A layer that displays map images obtained from map image data providers.
* [`OpenStreetMapDataProvider`](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.OpenStreetMapDataProvider) - A provider that loads map images in the OpenStreetMap format from a web resource.

    > **IMPORTANT:** Before you use OpenStreetMap tiles in your app, read the following articles: [Copyright and License](https://www.openstreetmap.org/copyright) and [Tile Usage Policy](https://operations.osmfoundation.org/policies/tiles). 

> * [`InformationLayer`](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.InformationLayer) - A layer that displays additional geo information above the map.

Reviewer: ```suggestion
* [`InformationLayer`](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.InformationLayer) - A layer that displays additional geo information over the map.
```
